lib/hotp/two-factor-authenticator.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
```

Comment:
This file should be describing what it does.  If we require 10 factors for authentication, what factor is this one?

I'm thinking we want to rename the file from `hotp/two-factor-authenticator.js` to `mfa/hotp.js`.  Imagine we have all of these MFA solutions:

* Code sent via text message - `sms.js`
* Physical token mailed to a user - `hardware-token.js`
* Another person validating that the first one is trying to get in - `trusted-local-authority.js`
* Password (yeah, passwords are a factor in "multi factor") - `password.js`
* Voice call to their cell phone - `voice-call.js`
* and this module... - `virtual-token.js`?